contacts/tests.py

```diff
@@ -1,12 +1,53 @@
 from django.test import TestCase
+
+from contacts.forms import AddContactForm
 from contacts.models import Contact, Number, Email, User
+from django.contrib.auth.models import User
 
 
 class ContactTestCase(TestCase):
+
+    def setUp(self):
+        user = User(username='sharukh', password='shahrukh31')
+        user.save()
+        Contact.objects.create(first_name="shahrukh", last_name="ijaz", note="mine",  dob='2019-03-14', user_id=user)
+
+    def able_to_delete_contact(self):
```

Comment:
Any method of any sub-class of `TestCase` which doesn't start with `test_` is not considered a test case. Please rename all such test cases as such.

Reply:
done